LLM_Compressor_0.9.0.md

```diff
@@ -0,0 +1,142 @@
+# LLM Compressor v0.9.0 release notes
+
+While working on this release, LLM Compressor reached over 100 contributors. Thank you to everyone for your contributions!
+
+This LLM Compressor v0.9.0 release adds:
+
+- Extended KV cache and attention quantization support
+- Batched calibration support
+- A new `model_free_ptq` pathway
+- A new _AutoRound_ modifier
+- Experimental support for MXFP4 quantization
+
+Additionally, this release finalizes the initial support work for SpinQuant R3 style transforms.
+This release also updates AWQ support to be further simplified and generic to quantization schemes beyond W4A16.
+
+<!-- https://issues.redhat.com/browse/INFERENG-2911 -->
+## Updated support for KV Cache and attention quantization ✨
+
+KV cache quantization has been refactored to address limitations in supported schemes and lifecycle-related bugs.
+The implementation has moved to compressed tensors, with new support for attention quantization.
+
+This refactoring enables the following updates:
+- Quantization of KV cache and attention using any scheme, including the new per-head strategy.
+- Running KV/attention quantization experiments within Hugging Face for research and accuracy validation.
+- Full compatibility with attention rotation via transforms.
+
+These changes lay the groundwork for future work in creating and researching advanced attention and KV-cache quantized models.
+
+<!-- https://issues.redhat.com/browse/INFERENG-3135 -->
+<!-- https://issues.redhat.com/browse/INFERENG-3136 -->
+## Added model-free post-training quantization ✨
+
+Model-free post-training quantization (PTQ) enables quantization by directly operating on safetensors files.
+This is particularly useful for models without a `transformers` model definition, such as some Mistral models.
+
+See the [model_free_ptq](https://github.com/vllm-project/llm-compressor/tree/main/examples/model_free_ptq) usage examples for more information.
+
+## Added AutoRound modifier
+
+Added [AutoRoundModifier](https://github.com/vllm-project/llm-compressor/blob/main/examples/autoround/llama3_example.py) for quantization that uses [AutoRound](https://aclanthology.org/2024.findings-emnlp.662.pdf), an advanced post-training algorithm that optimizes rounding and clipping ranges through sign-gradient descent.
+This approach combines the efficiency of post-training quantization with the adaptability of parameter tuning, delivering robust compression for large language models while maintaining strong performance.
+
+<!-- https://issues.redhat.com/browse/INFERENG-2542 -->
+## Calibration performance improvements
+
+LLM Compressor now supports batched calibration for quantization and sparsification.
+Pass the `batch_size` and `data_collator` arguments to the `oneshot` compression entrypoint for improved calibration throughput.
+For built-in collation strategies, pass `"padding"` or `"truncation"` as string values for `data_collator`.
```

Comment:
Can we add one line here:
`This updates the default collator from DefaultDataCollator to `truncation` and also updates the default value for `shuffle_calibration_samples` from True to False